tests/copier_data/data2.yaml

```
@@ -5,11 +5,12 @@ is_open_source: false
 description: Doing crazy things! So many things!
 install_claude_cli: true
 ssh_port_number: 54321
+pull_from_ecr: true
 
 
 
 
```

YAMLlint (1.38.0)

[error] too many blank lines (4 > 2)

(empty-lines)